main.js

```diff
@@ -8,32 +7,38 @@ var bodyInput = document.querySelector('.body-input');
 var searchIdeasInput = document.querySelector('.search-ideas-box');
 var titleInput = document.querySelector('.title-input');
 var displaySection = document.querySelector('.idea-display-section');
+var closeCardBtn = document.querySelector('.close-card');
+var star = document.querySelector('.star');
 var ideasList = [];
 var newIdea;
 
 //-----------Event Listeners----------//
-
 // filterStarIdeaBtn.addEventListener('', );
 saveIdeaBtn.addEventListener('click', function() {
 createIdea(event);
 });
 // showStarIdeaBtn.addEventListener('', );
-// bodyInput.addEventListener('', );
 // searchIdeasInput.addEventListener('', );
-// titleInput.addEventListener('', );
 bodyInput.addEventListener('keyup', disableSaveButton);
 titleInput.addEventListener('keyup', disableSaveButton);
 window.addEventListener('load', renderIdea);
+displaySection.addEventListener('click', function(event) {
+  if (event.target.className === 'close-card') {
+    event.target.closest('article').remove();
+  }
+});
 
-//-------------functions----------------//
+displaySection.addEventListener('click', function(event) {
```

Comment:
very clever! Next session we should move this up into our event listener section.